prow/cmd/deck/tide.go

```diff
@@ -155,60 +159,103 @@ func (ta *tideAgent) updateHistory() error {
 	return nil
 }
 
-func (ta *tideAgent) filterHiddenPools(pools []tide.Pool) []tide.Pool {
-	if len(ta.hiddenRepos()) == 0 {
-		return pools
-	}
+func (ta *tideAgent) matchingIDs(ids []string) bool {
+	return len(ids) > 0 && len(ta.tenantIDs) > 0 && sets.String{}.Insert(ta.tenantIDs...).HasAll(ids...)
+}
 
+func (ta *tideAgent) filterHiddenPools(pools []tide.Pool) []tide.Pool {
```

Comment:
nit, the name of the function is not accurate any more, it also filters based on tenantIDs, not necessarily hidden